utilities/test_suite/rpp_test_suite_common.h

```diff
@@ -41,15 +41,6 @@ SOFTWARE.
 #include <random>
 #include <map>
 
-#ifdef GPU_SUPPORT
-    #include <hip/hip_fp16.h>
-#else
-    #include <half/half.hpp>
```

Comment:
This has been removed in the old and new test suite both, and the library code. 
- Have all builds been checked on all backends?
- Then possibly run one HIP function say brightness on F16 specifically to confirm visual unit test output since we don't have QA test for F16.

Please confirm here if its good.